client/my-sites/checkout/composite-checkout/components/jetpack-pro-redirect-modal/index.tsx

```diff
@@ -0,0 +1,111 @@
+import { Button } from '@automattic/components';
+import { Modal } from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+import { CheckIcon } from 'calypso/../packages/composite-checkout/src/components/shared-icons';
```

Comment:
This should be a package import. We should not reach inside the private implementation details of a package. 

```suggestion
import { CheckIcon } from '@automattic/composite-checkout';
```

Reply:
I think this happened because of auto-import. I agree with this and fixed it here - https://github.com/Automattic/wp-calypso/pull/78955/commits/6ebd952eeb69ec8e94332682268322499ad30496

Reply:
Yeah, that happens to me all the time too. I should check with the Calypso team to see if there's a way to add a rule to catch these sorts of imports. 🤔 @jsnajdr do you think that would be possible and/or would it be too much work?

Reply:
> see if there's a way to add a rule to catch these sorts of imports.

You mean an eslint rule? Maybe `eslint-plugin-import` has some rule for this out of the box. If we wanted to write our own rule, it shouldn't be terribly hard. We'd mainly have to specify what exactly are the imports we want to avoid.

The auto-import suggestions are created by VSCode, which in turn uses the JS Language Server to generate them.

Reply:
>  We'd mainly have to specify what exactly are the imports we want to avoid.

I think that it would be safe to say that we want to block all imports that try to reach a path starting with `calypso/../packages`. To also block relative imports we could say any path including `../packages` but it is possible that could find false positives.